tree_sitter_v/.gitignore

```
@@ -3,6 +3,7 @@ node_modules/
 *.log
 build/
 package-lock.json
+package.swift
```

```suggestion
Package.swift
```
It's capital P with the package swift.  Afaik `.gitignore` is case sensitive. On my end when it's generated with the new tree-sitter cli versions it's not ignored.